src/pages/video-test.tsx

```diff
@@ -78,7 +81,21 @@ const EggheadPlayer: React.FC<{videoResource: VideoResource}> = ({
                 kind="metadata"
                 label="notes"
               />
-              <CueBar key="cue-bar" order={6.0} />
+              <CueBar
+                onClickCue={() => {
+                  playerPrefs.setPlayerPrefs({sideBar: {activeTab: 0}})
+                  setTimeout(() => {
+                    scroller.scrollTo('active-note', {
+                      duration: 0,
+                      delay: 0,
+                      offset: -12,
+                      containerId: 'notes-tab-scroll-container',
+                    })
+                  }, 0.1)
```

Comment:
Clicking cue:
1. activates `Notes` tab
2. scrolls down to active note.

I couldn't get it working properly in all scenarios until I wrapped scroll function inside very short `setTimeout`. I guess that it's attempting to scroll sooner than `NotesTabContent` exist resulting in no scroll at all. Feels like a hack. 😭